Marlin/SanityCheck.h

```diff
@@ -64,11 +64,12 @@
       #error EXTRUDERS must be 1 with HEATERS_PARALLEL.
     #endif
 
-    #ifdef Y_DUAL_STEPPER_DRIVERS
+    // Dual axis only when less then two extruders defined and an second extruder is available
+    #if ((EXTRUDERS>2) || (defined(Y_DUAL_STEPPER_DRIVERS) && (!defined(E2_STEP_PIN))))
```

Comment:
Should there be a separate error for `defined(Y_DUAL_STEPPER_DRIVERS) && defined(E2_STEP_PIN)`, stating that they can't both be defined at once?

This `EXTRUDERS > 2` condition will always be triggered when `EXTRUDERS > 2` so it doesn't belong here. This section is also only evaluated when `EXTRUDERS > 1` so that condition doesn't need to be checked here either.